packages/blockchain-wallet-v4/src/network/api/eth/index.ts

```diff
@@ -7,7 +7,7 @@ export default ({ apiUrl, get, openseaApi, post }) => {
   const IS_TESTNET = openseaApi && openseaApi.includes('rinkeby')
 
   const ethProvider = IS_TESTNET
-    ? new ethers.providers.EtherscanProvider('rinkeby')
+    ? new ethers.providers.EtherscanProvider('rinkeby', 'X4TZ5S2F3MPD3TBTMZ3JDGVKC79472VPPC')
```

Comment:
Nothing to see here, never saw this key :eyes:

Reply:
😨

Reply:
Its a test key, Ill regen :)